.github/workflows/snyk-scan.yml

```diff
@@ -17,14 +17,46 @@ jobs:
     name: All SDKs images
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

Comment:
I suspect there's a good reason for this -- worth documenting?

Reply:
Yeah we want to complete as many scans as possible, if one fails for some reason. I added a comment.